Scripts/Mend SCA/get-malicious-packages.sh

```diff
@@ -1,25 +1,25 @@
 #!/bin/bash
 # Prerequisites:
-# apt install jq curl xargs 
+# apt install jq curl
 # WS_USERKEY - An administrator's userkey
```

Comment:
Use MEND variables as defined in the [CLI documentation](https://docs.mend.io/bundle/integrations/page/authenticate_your_login_for_the_mend_cli.html#Mend-CLI---mend-login-parameters)